### PR TITLE
Update SHACL docs for 3.1 release

### PR DESCRIPTION
**Pull request by hmottestad**

### CODE REVIEW

**Comment by hmottestad** [comment]
@jeenbroekstra I've updated the docs, to be fair I think most of these updates actually affect 3.0.0....but it'll be nice to get this out together with the 3.1.0 release.

File: site/content/documentation/programming/shacl/index.md [hunk 1]
```diff
@@ -14,125 +14,300 @@ This documentation is for RDF4J 2.5 and onwards, as the SHACL engine was experim
...
 
-For this example the SHACL engine will match the predicate `ex:age` with `ex:PersonShape` and realise that `ex:pete` might already be defined as a `ex:Person` in the database. The validation plan will then include checking if `ex:pete` is type `ex:Person`.
+# How to load and update SHACL shapes
 
-# How to load SHACL shapes
+The ShaclSail uses a reserved graph (`http://rdf4j.org/schema/rdf4j#SHACLShapeGraph`) for storing the SHACL shapes. 
+Utilize a normal connection to load your shapes into this graph. SPARQL is not supported.
 
-The ShaclSail uses a reserved graph (`http://rdf4j.org/schema/rdf4j#SHACLShapeGraph`) for storing the SHACL shapes. Utilize a normal connection to load your shapes into this graph. 
+```java
```
Comment (abrokenjester) [inline site/content/documentation/programming/shacl/index.md]: This doesn't work as expected. It formats as a preformatted code block but uses no syntax highlighting. You need to use the `{{< highlight java >}}` shortcode instead for that.

**Review by abrokenjester — COMMENTED** [review]
Looks good, though one inline comment regarding code formatting.

File: site/content/documentation/programming/shacl/index.md (hunk 1 above)
Comment (hmottestad) [reply site/content/documentation/programming/shacl/index.md]: What processor are we using for our markdown? The {{}} syntax isn’t recognized by any or the ones I am familiar with.

Comment (barthanssens) [reply site/content/documentation/programming/shacl/index.md]: We use Hugo to generate the docs on the website (https://gohugo.io/templates/introduction/)

Comment (hmottestad) [reply site/content/documentation/programming/shacl/index.md]: It looks like it should be working in 0.60.0 https://gohugo.io/content-management/syntax-highlighting/#highlighting-in-code-fences

Comment (abrokenjester) [reply site/content/documentation/programming/shacl/index.md]: I can have a look at upgrading our hugo instance. Logged as https://github.com/eclipse/rdf4j/issues/1902

Comment (abrokenjester) [reply site/content/documentation/programming/shacl/index.md]: Unfortunately hugo 0.60 has several non-compatible changes, I want to take a good look at those before upgrading.

There's also the fact that we use the hugo shortcode everywhere else. The triple backtick block gets rendered different from our other code blocks. I'll push a fix to this PR to use the old syntax for now so we can merge this, and then we can start looking at upgrading to a newer hugo version as a separate issue.

**Review by abrokenjester — APPROVED** [review]

**Pull request merged**